Google.Api.Gax.Grpc/Rest/RestChannel.cs

```diff
@@ -59,7 +59,12 @@ public RestChannel(RestServiceCollection serviceCollection, string endpoint, Cha
         {
             var restMethod = _serviceCollection.GetRestMethod(method);
 
-            var cancellationTokenSource = new CancellationTokenSource();
+            CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
```

Comment:
Not sure why you've changed the `var` to an explicit type here... I think it's clear enough as it is.